core/src/com/fallenflame/game/GameEngine.java

```diff
@@ -253,6 +264,11 @@ public boolean preUpdate(float dt) {
             listener.exitScreen(this, EXIT_QUIT);
             return false;
         }
+        else if (countdown > 0) {
+            countdown--;
+        } else if (countdown == 0) {
+            reset();
+        }
```

Comment:
Why are we resetting if countdown is reached?

Reply:
Countdown is -1 until we win or lose. It's used purely to show "Victory" or "You Died" for a certain amount of time (ie until countdown reaches 0). Once it reaches 0, reset